### ISSUE TITLE
BUG: Error is thrown to user not member of private community, when send message

### ISSUE THREAD

**Issue by Comoque1** [cherrytwist/Server#1521]
**Describe the bug**
Error is thrown to user not member of community, when send message.

**To Reproduce**
1. Go to https://dev.alkem.io/graphql
2. Authenticate with any user not member of a community (i.e.)
```
mutation sendMessageToDiscussion($messageData: DiscussionSendMessageInput!) {
  sendMessageToDiscussion(messageData: $messageData) {
    id
  }
}
```
```
{
  "messageData": {
    "message": "example222",
    "discussionID": "44e97547-2e0b-4dc3-80ab-91f22c2330d5"
  }
} 
```
3. After performing the mutation error is thrown: ` "message": "User @newac=test.com:matrix-dev.alkem.io not in room !kUrOCEgrIjuDifjQgI:matrix-dev.alkem.io (<FrozenEventV3 event_id='$me7GabnfHKIEt8LZKRsWhujuUXahDhrhjyfGilJBVTY', type='m.room.member', state_key='@newac=test.com:matrix-dev.alkem.io'>)",`
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/fd8fa31b-e63f-4070-9c04-a0da58633493)

**Expected behavior**
Error thrown must coming from the one defined in the server (i.e.` "message": "Authorization: unable to grant 'create' privilege: discussion send message: bb",` )

## Note:
When the hub is public, non ecoverse member can create discussion and can send messages

**Comment by techsmyth** [comment]
This may also be fixed by #1520. Can you please re-test?

**Comment by Comoque1** [comment]
Tested OK on Test env, with the branch mentioned by @techsmyth.
Closing.
